tungsten_tempest_plugin/tests/api/contrail/rbac_base.py

```diff
@@ -395,6 +392,9 @@ def setup_clients(cls):
     @classmethod
     def resource_setup(cls):
         cls.tenant_name = cls.os_primary.credentials.tenant_name
+        cls.tenant_id = cls.fq_client.fqname_to_id(fq_name=['default-domain',
```

Comment:
IMO usage of `cls.get_tenant_by_name(tenant_name)` instead of `cls.fq_client`  is better.

Reply:
It doesn't look like get_tenant_by_name will make the necessary call to the TF API to make it pull the information from Keystone.

Reply:
Ok, I see, but what tests use the tenant_id variable?  What is a reason of having this variable?

Reply:
The point is just to make the API call. If we don't do this, subsequent tests that rely on the creation of project-scoped resources (such as virtual-networks) fail.

The return value could be discarded or ignored instead, but it seemed to make sense to keep it around in case a future test wants it.